ansible_collections/arista/cvp/plugins/module_utils/device_tools.py

```diff
@@ -1739,6 +1780,118 @@ def reset_device(self, user_inventory: DeviceInventory):
             results.append(result_data)
         return results
 
+    def validate_config(
+        self,
+        user_inventory: DeviceInventory,
+        validate_mode: str = ModuleOptionValues.VALIDATE_MODE_SKIP,
+    ):
+        results = []
+        device_data = {"warnings": [], "errors": []}
+        vldm_skip = ModuleOptionValues.VALIDATE_MODE_SKIP
```

Comment:
if the mode is skip in the first place why are we running the code? it looks quite inefficient

or maybe the name of the option is not the best - maybe use like in YANG continue-on-error or whatever 

if I choose skip I would expect the validation not to run

Reply:
Also maybe let's import the constants and not declare variables to map to constants

Reply:
we want to validate always, skip just means we don't want ansible to fail the playbook, ie skip to next task
sure I'll import the constants, but what if the line is too long? we still don't use max line length 160 sadly